saleor/userprofile/utils.py

```diff
@@ -14,3 +14,15 @@ def store_user_address(user, address, billing=False, shipping=False):
     if changed:
         user.save()
     return entry
+
+
+def get_customers(request):
```

Comment:
This name is not very descriptive. Something like `get_users_to_impersonate` would be much better. Since this function and `can_impersonate` below are used only by the `impersonate` module, they're not really utility functions that could be reused somewhere else (due to the `request` argument). I would consider moving them to a separate module e.g. `userprofile/impersonate.py`.